src/rp2_common/hardware_pio/include/hardware/pio.h

```diff
@@ -723,11 +723,12 @@ static inline void pio_set_irq1_source_enabled(PIO pio, enum pio_interrupt_sourc
         hw_clear_bits(&pio->inte1, 1u << source);
 }
 
-/*! \brief  Enable/Disable multiple sources on a PIO's IRQ 0
+/*! \brief  Enable/Disable multiple source on a PIO's IRQ 0
  *  \ingroup hardware_pio
  *
+ * \param pio The PIO instance; either \ref pio0 or \ref pio1
  * \param source_mask Mask of bits, one for each source number (see \ref pio_interrupt_source) to affect
- * \param enabled true to enable all the source specified in the mask on IRQ 0, false to disable all the source specified in the mask on IRQ 0
+ * \param enabled true to enable all the source specified in the mask on IRQ 0, false to disable all the sources specified in the mask on IRQ 0
```

Comment:
You've fixed the second "all the source" -> "all the sources", but missed the first "all the source" (also on this same line).

Reply:
fxd